docs/source/Evaluation_Tutorial.md

```diff
@@ -177,11 +111,12 @@ You have to specify run parameters required by this pipeline:
 - `checkpoint_dir`: Path to store output Estimator checkpoints. You can use one of the checkpoints for estimator evaluation.
 - `volume_size`: Size of the Kubernetes Persistent Volume Claims (PVC) that will be used to store the dataset. You should change this value according to the dataset that was generated. If you use default settings from [these instructions](https://github.com/Unity-Technologies/SynthDet/blob/master/docs/RunningSynthDetCloud.md), you should expect `1.2TiB` storage required for 400k images.
 
-> You'll want to change `tb_log_dir`, `checkpoint_dir` to point to a location that is convenient for you and your Kubernetes cluster have permissions to write to. This is typically a GCS path under the same GCP project. You want to keep a note on these directories that will be used for tensorboard visualization and model [evaluation](#part-3-evaluate-a-model). Note that an invalid location will cause the job to fail, whereas a path to the local filesystem may run but will be hard to monitor as you won't have easy access to these files inside a docker container.
+Set `tb_log_dir` and `checkpoint_dir` to a location that is convenient for you and your Kubernetes cluster has permissions to write to. This is typically a GCS path under the same GCP project. You want to keep a note on these directories that will be used for tensorboard visualization and model evaluation. Note that an invalid location will cause the job to fail, whereas a path to the local filesystem may run but will be hard to monitor as you won't have easy access to the files.
 
 ![pipeline graph](_images/kubeflow/train_pipeline_graph.jpg)
 
-To open tensorboard for training visualization, you can run the following command:
+## Part 3: Monitor training in tensorboard
```

Comment:
+100 for this change. This makes it easier to reduce visual noise of tensorboard monitoring that was repeated in multiple sections.